python/ql/test/experimental/library-tests/frameworks/stdlib/SafeAccessCheck.py

```diff
@@ -0,0 +1,8 @@
+s = "taintedString"
+
+if s.startswith("tainted"):  # $checks=s $branch=true
+    pass
+
+sw = s.startswith  # $f-:checks=s $f-:branch=true
+if sw("safe"):
```

Comment:
Wouldn't we under ideal circumstances expect this comment on line 7 instead?

Reply:
yes..

Reply:
Let's leave it for later. We'll want to revisit this when we fix it anyway.

Reply:
true